File: OpenTabs.py
import webbrowser
import time

class OpenTabs:
    def __init__(self, urls):
        self.urls = urls        

    def open_all(self):
        for url in self.urls:
            webbrowser.open_new_tab(url)
            time.sleep(1)   
        print("All tabs opened.")

if __name__ == "__main__":
    urls = [
        "https://mail.google.com/mail/u/0/#inbox",
        "https://leetcode.com/problemset/",
        "https://chatgpt.com/",
        "https://www.w3schools.com/",
        "https://www.youtube.com/"
    ]
    opener = OpenTabs(urls)
    opener.open_all()

File: OpenTabs.java
import java.awt.Desktop;
import java.net.URI;

public class OpenTabs {
    public static void main(String[] args) {
        String[] urls = {
                "https://mail.google.com/mail/u/0/#inbox",
                "https://leetcode.com/problemset/",
                "https://chatgpt.com/",
                "https://www.w3schools.com/",
                "https://www.youtube.com/"
        };

        try {
            Desktop desktop = Desktop.getDesktop();
            for (String url : urls) {
                desktop.browse(new URI(url));
                Thread.sleep(1000); // Pause for 1 second between opening tabs
            }
            System.out.println("All tabs opened successfully.");
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}